ground/gcs/src/plugins/systemhealth/systemhealthgadgetwidget.cpp

```diff
@@ -138,7 +138,6 @@ SystemHealthGadgetWidget::~SystemHealthGadgetWidget()
 
 void SystemHealthGadgetWidget::setSystemFile(QString dfn)
 {
-    setBackgroundBrush(QBrush(Utils::StyleHelper::baseColor()));
```

Comment:
You are defeating the setting on the main GCS settings page here. I think we should either remove the setting and force it everywhere, or leave it as-is and change the color in the default configs rather than end up with a hodge-podge of different things.